src/tests/testIpAddress.cc

```diff
@@ -8,6 +8,8 @@
 
 #include "squid.h"
 #include "compat/cppunit.h"
+#include "compat/netdb.h"
+#include "compat/socket.h"
```

Comment:
API provided by this include is not used by this .cc file.

Reply:
removed